API/Features/Customer/Requests/DeletePortalUser.cs

```diff
@@ -0,0 +1,63 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using DLCS.Core.Collections;
+using DLCS.Repository;
+using MediatR;
+
+namespace API.Features.Customer.Requests;
+
+public class DeletePortalUser : IRequest<DeletePortalUserResult>
+{
+    public DeletePortalUser(int customerId, string userId)
+    {
+        CustomerId = customerId;
+        UserId = userId;
+    }
+
+    public int CustomerId { get; }
+    public string UserId { get; }
+    
+}
+
+
+public class DeletePortalUserResult
+{
+    public string Error { get; set; }
+}
+
+public class DeletePortalUserHandler : IRequestHandler<DeletePortalUser, DeletePortalUserResult>
+{
+    private readonly DlcsContext dbContext;
+
+
+    public DeletePortalUserHandler(
+        DlcsContext dbContext)
+    {
+        this.dbContext = dbContext;
+    }
+
+
+    public async Task<DeletePortalUserResult> Handle(DeletePortalUser request, CancellationToken cancellationToken)
+    {
+        var dbUser = await dbContext.Users.FindAsync(new object?[]{request.UserId}, cancellationToken);
+        if (dbUser == null)
+        {
+            return new DeletePortalUserResult { Error = "No such user" };
+        }
+
+        if (dbUser.Customer != request.CustomerId)
+        {
+            return new DeletePortalUserResult { Error = "User doesn't belong to customer" };
```

Comment:
Is it risky to return this information as it highlights which emails are in the system? Maybe log a message but return something less detailed?